src/ESPEasy-Globals.h

```diff
@@ -176,7 +176,7 @@ void check_size() {
 #include <map>
 #include <deque>
 
-#include "src/DataStructs/ESPEasyLimits.h"
+//SMY #include "src/DataStructs/ESPEasyLimits.h" //already included
```

Comment:
Multiple includes of the same header file should not be an issue.
That's why I wrapped the content of each header file in #ifndef ... #define... #endif

Reply:
Ah, never mind, I see now it is already in the same file included :)

Reply:
agree, but it's some 20lines futher up where it's included, so it more of a cleanup...